app/assets/javascripts/discourse/models/composer.js.es6

```diff
@@ -912,7 +912,7 @@ const Composer = RestModel.extend({
   },
 
   createPost(opts) {
-    if (this.action === CREATE_TOPIC) {
+    if (CREATE_TOPIC === this.action || PRIVATE_MESSAGE === this.action) {
```

Comment:
PSA: when sending the information to the server, we need to make sure the "topic_id" is [null/blank](https://github.com/discourse/discourse/blob/31f3ed8d3610b63c7b1146e1a67ee51b703dbc41/lib/post_creator.rb#L593-L595), otherwise, the server will think it's a reply instead of a new topic/message.